docs/src/main/paradox/docs/fusion/studio.md

```diff
@@ -126,13 +126,13 @@ An example `Dashboard` resource looks like this:
 }
 ```
 
-The `Dashboard`, once configured, will present the results of your sparql query in a table. Each item in the row will be clickable and lead to a details page showing the `Resource`, in this case, a list of scientists that have contributed data to the project.
+The `Dashboard`, once configured, will present the results of your SPARQL query in a table. Each item in the row will be clickable and lead to a details page showing the `Resource`, in this case, a list of scientists that have contributed data to the project.
 
 ## How to Configure a Studio
 
 ### Studio Creation
 
-To create a new studio, go to the Admin space, and select an organization and a project.
+To create a new `Studio`, go to the Admin space, and select an organization and a project.
```

Comment:
In the new interface, you need to click on Manage Studios or something in the top right corner.